sdcm/utils/decode_kallsyms.py

```diff
@@ -0,0 +1,48 @@
+import bisect
+import re
+import subprocess
+
+
+def binary_search_recursive(array, element):
+    return bisect.bisect_right(array, element, key=lambda y: int(y.split()[0], 16)) - 1
+
+
+
```

Comment:
Redundant line here and at the end of the file.

Reply:
removed